src/pages/inventory/index.tsx

```diff
@@ -0,0 +1,265 @@
+      handleFilter()
+    } else {
+      fetchData();
+    }
+  }, [type, category, status, search])
+
+
+  return (
+    <Box>
+
+      {/* Add button */}
+      <Box id="add-container" sx={{ display: 'flex', justifyContent: 'end' }}>
+        <Button sx={{ bgcolor: '#F5F5F5', color: 'black' }}><AddIcon fontSize="small" sx={{ color: 'black' }} />Add</Button>
+      </Box>
+
+      {/* Filter Container */}
+      <Box id="filter-container" sx={{ display: 'flex', alignItems: 'center', maxWidth: '90%' }}>
+        <Typography variant="body2">Filters</Typography>
+
+        {/* Type Filter */}
+        <Box sx={{ px: '8px' }} id="type-button-container">
+          <Button sx={{ color: 'black', bgcolor: '#E0E0E0', height: '30px' }} onClick={handleTypeClick}>
+            {type ? (
+              <>{type} <ClearIcon sx={{ fontSize: 'large', ml: '6px' }} onClick={clearTypeFilter} /></>
+            ) : (<><Typography variant="body2">Type</Typography><ExpandMoreIcon sx={{ fontSize: 'large', ml: '6px' }} /></>)}
+          </Button>
+          <Menu
+            open={Boolean(anchorType)}
+            onClose={handleTypeClose}
+            anchorEl={anchorType}
+          >
+            <MenuItem onClick={() => handleMenuTypeClick('Donation')}>Donation</MenuItem>
+            <MenuItem onClick={() => handleMenuTypeClick('Welcome Basket')}>Welcome Basket</MenuItem>
+          </Menu>
+        </Box>
+
+        {/* Category Filter */}
+        <Box sx={{ px: '8px' }} id="category-button-container">
+          <Button sx={{ color: 'black', bgcolor: '#E0E0E0', height: '30px' }} onClick={handleCategoryClick}>       {category ? (
+            <>{category} <ClearIcon sx={{ fontSize: 'large', ml: '6px' }} onClick={clearCategoryFilter} /></>
+          ) : (<><Typography variant="body2">Category</Typography><ExpandMoreIcon sx={{ fontSize: 'large', ml: '6px' }} /></>)}</Button>
+          <Menu
+            open={Boolean(anchorCategory)}
+            onClose={handleCategoryClose}
+            anchorEl={anchorCategory}
+          >
+            <MenuItem onClick={() => handleMenuCategoryClick('Beddings & Linens')}>Beddings & Linens</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Bath & Toiletries')}>Bath & Toiletries</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Kitchenware')}>Kitchenware</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Decorative & Home improvement')}>Decorative & Home improvement</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Health & Medical')}>Health & Medical</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Food')}>Food</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Electronics & Appliances')}>Electronics & Appliances</MenuItem>
+            <MenuItem onClick={() => handleMenuCategoryClick('Games, Books, Entertainment')}>Games, Books, Entertainment</MenuItem>
+          </Menu>
```

Comment:
Consider that the application allows users to create new categories, the hard coding way might limit the flexibility for future changes.

Reply:
I think we could add this functionality into an admin page. I can check with with Bai, her design did not call for adding category/types

Reply:
We discussed this briefly with PH this week on site. I agree with Anthony. Let's stick with hard coding for now. It should be in an admin page if anywhere.